luaui/Widgets_BAR/cmd_select_commander.lua

```diff
@@ -0,0 +1,28 @@
+function widget:GetInfo()
+  return {
+    name = "SelectCommander",
+    desc = "Press ctrl+c to select the commander",
+    author = "Teifion",
+    date = "26/10/2020",
+    license = "GNU GPL, v2 or later",
+    layer = 0,
+    enabled = true
+  }
+end
+
+function widget:Update()
+  local t = Spring.GetGameSeconds()
+  if Spring.GetSpectatingState() then
+    widgetHandler:RemoveWidget(self)
+    return
+  end
+  if (t > 0) then
```

Comment:
I think this check doesn't do anything meaningful. You don't have any commanders at `t <= 0` anyway, and if you do (via `/give` or a map gadget perhaps) it may well be fine to let them be selected with Ctrl+C.

Reply:
To test, type these ingame:
/cheat
/give armcom
/give armdecom